funnel/models/commentvote.py

```diff
@@ -95,7 +98,9 @@ class Comment(BaseMixin, db.Model):
 
     message = MarkdownColumn('message', nullable=False)
 
-    status = db.Column(db.Integer, default=0, nullable=False)
+    _state = db.Column('status', db.Integer, StateManager.check_constraint('status', COMMENTSTATUS),
```

Comment:
If you're adding a CHECK constraint, you need a corresponding migration.

Reply:
`db migrate` doesn't detect anything. does it have to be manually added?

Reply:
Yes. Alembic's auto-detection isn't comprehensive.